jwst/datamodels/schemas/flat.schema.yaml

```diff
@@ -7,6 +7,10 @@ allOf:
 - $ref: subarray.schema
 - $ref: keyword_filter.schema
 - $ref: keyword_pfilter.schema
+- $ref: keyword_grating.schema
```

Comment:
If grating and pgrating are for NIRSpec, I don't think they belong in this schema. NIRSpec uses the dflat, fflat, and sflat schemas, not the generic "flat" schema.

Reply:
https://github.com/spacetelescope/jwst/blob/master/jwst/datamodels/schemas/nirspec_flat.schema.yaml#L8

Reply:
removed

Reply:
And added `band` for MIRI as I see it in the selector